ui/src/globals/store.js

```diff
@@ -1,6 +1,7 @@
-import Vue from "vue";
+// import Vue from "vue";
```

Comment:
lets clean this up

Reply:
thanks for pointing out some obsolete mapbox references. There are still some more left, but these will be a topic for another commit.